plutus-core/plutus-core/src/PlutusCore/Builtin/Runtime.hs

```diff
@@ -8,8 +8,6 @@
 
 module PlutusCore.Builtin.Runtime where
 
-import PlutusPrelude
-
```

Comment:
Hmm... That's a weird change!

Reply:
I don't understand this change. I don't anything from `PlutusPrelude` removed from the module, so it must have already been useless? How come GHC didn't complain about `PlutusPrelude` being useless?

Reply:
Just checked, it is useless on `master` and GHC doesn't complain! How so?

Reply:
@thealmarty could you try adding the useless `PlutusPrelude` import back and running the benchmarks?

Reply:
I really hope it's not GHC deciding the import is not redundant, because its name ends in `Prelude`.